local/nginx/include/content.conf

```diff
@@ -38,23 +38,21 @@ location ~ ^/content/contents/([^\/]+)/active-entities$ {
 
     proxy_pass http://content-server/contents/$1/active-entities$is_args$args;
 }
-
-# Tier C
-location ~ ^/content/contents/([^\/]+)$ {
+location ~ ^/content/entities$ {
     include /etc/nginx/include/content_proxy.conf;
 
-    limit_req zone=tier_c burst=50 nodelay;
+    limit_req zone=tier_b burst=8 nodelay;
 
-    proxy_pass http://content-server/contents/$1$is_args$args;
+    proxy_pass http://content-server/entities$is_args$args;
 }
 
-# Tier D
-location ~ ^/content/entities$ {
+# Tier C
+location ~ ^/content/contents/([^\/]+)$ {
```

Comment:
This endpoint may not be rate limited

Reply:
This is the current config: 
limit_req_zone $user_identifier zone=tier_c:10m rate=1000r/m;